framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -188,24 +200,24 @@ public Postcondition(
          * {@code foo} is guaranteed to be {@code @NonNull} after a call to {@code method()} if that
          * call returns {@code false}.
          */
-        public final boolean annoResult;
+        public final boolean resultValue;
```

Comment:
This is a breaking refactor.  I might be worth reverting just this renaming so me can merge the rest of the changes.